Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -52,6 +77,22 @@ void main (void)
     
     vec3 rgb = rayleighPhase * v_rayleighColor + miePhase * v_mieColor;
     rgb = vec3(1.0) - exp(-exposure * rgb);
+    // compute luminance before color correction to avoid strangely gray night skies
```

Comment:
Usually we capitalize the first word in comments. You may need to change one line in `SkyAtmosphereVS.glsl` as well.